docs/content/en/docs/installation/metrics.md

```diff
@@ -92,27 +100,50 @@ tetragon:
     metricsLabelFilter: "namespace,workload,binary" # "pod" label is disabled
 ```
 
-## Scrape metrics
+## Enable Prometheus ServiceMonitors
 
 Typically, metrics are scraped by Prometheus or another compatible agent (for example OpenTelemetry Collector), stored
 in Prometheus or another compatible database, then queried and visualized for example using Grafana.
 
-In Kubernetes, you can install Prometheus and Grafana using the `kube-prometheus-stack` Helm chart:
+In Kubernetes, you can install Prometheus and Grafana using the [Kube-Prometheus-Stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) Helm chart. This Helm chart includes [Prometheus Operator](https://prometheus-operator.dev/),
+which allows you to configure Prometheus via Kubernetes custom resources.
 
-```shell
-helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
-helm install kube-prometheus-stack prometheus-community/kube-prometheus-stack \
-  --namespace monitoring  --create-namespace \
-  --set prometheus.prometheusSpec.serviceMonitorSelectorNilUsesHelmValues=false
-```
-
-The `kube-prometheus-stack` Helm chart includes [Prometheus Operator](https://prometheus-operator.dev/), which allows
-you to configure Prometheus via Kubernetes custom resources. Tetragon comes with a default `ServiceMonitor` resource
-containing the scrape confguration. You can enable it via Helm values:
+Simultaneously, Tetragon comes with default `ServiceMonitor` resources containing the scrape confguration for the Agent
+and Operator. You can enable it via Helm values:
 
 ```yaml
 tetragon:
   prometheus:
     serviceMonitor:
       enabled: true
+tetragonOperator:
+  prometheus:
+    serviceMonitor:
+      enabled: true
 ```
+
+{{< warning >}}
```

Comment:
Personally I would make it a note not a warning, and maybe put the `helm install` command in the paragraph, only explain why `--set prometheus.prometheusSpec.serviceMonitorSelectorNilUsesHelmValues=false` is needed in a note, but that's a non-blocking preference.

Reply:
Makes sense. I changed it in the latest commit.